jaeger-core/src/main/java/com/uber/jaeger/Span.java

```diff
@@ -259,4 +271,45 @@ public Span log(long timestampMicroseconds, String message, /* @Nullable */ Obje
       return this;
     }
   }
+
+  /**
+   * Creates logs related to logged exception
+   *
+   * @param fields map containing exception logs which are not present in fields
+   * @return logged fields
+   */
+  private static Map<String, Object> exceptionLogs(Map<String, ?> fields) {
+    Object ex = fields.get("error.object");
```

Comment:
They are defined in the spec, but not in API artifact, I will open PR to ot-java.